TACC/cms-site-resources
# FP-947: Add 3Dem as project, include custom assets

## Overview

Add 3Dem project, along with custom assets.

## Related Tickets

* [FP-947](https://jira.tacc.utexas.edu/browse/FP-947)

## Summary of Changes

- __New__: `/3dem-org` directory and files (secrets diff, sample css structure, favicon, logo, template)

## Testing Steps

1. Follow [Wiki: Test New CMS Site](https://github.com/TACC/cms-site-resources/wiki/Test-New-CMS-Site).
    - The project name is `3dem-org`.
    - The remote docker image is `taccwma/core-cms:fa75889`.
    
    _You may ask @tacc-wbomar for assistance._

2. Ensure 3Dem favicon is loaded from `3dem-org` file path.
2. Ensure 3Dem logo is loaded from `3dem-org` file path.

## UI Photos

![FP-947](https://user-images.githubusercontent.com/62723358/111525922-196fcc00-872c-11eb-82e8-33569591535b.png)

## Notes
